pkg/agent/openflow/client.go

```diff
@@ -580,6 +628,9 @@ func (c *client) InstallGatewayFlows() error {
 	if c.encapMode.SupportsEncap() {
 		flows = append(flows, c.l3FwdFlowToGateway(gatewayIPs, gatewayConfig.MAC, cookie.Default)...)
 	}
+	if c.encapMode.SupportsNoEncap() {
+		flows = append(flows, c.getGatewayFlowsNoEncap(gatewayConfig)...)
```

Comment:
It seems the flows are exactly same as `l3FwdFlowToGateway`, why don't just add one more condition to above if check?

Reply:
Updated.